the-blue-alliance-ios/AppDelegate.swift

```diff
@@ -146,7 +99,60 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     // MARK: Private
 
-    func setupAppearance() {
+    private func rootSplitViewController() -> UISplitViewController {
```

Comment:
Let's create this, the tab bar, etc. in code (as opposed to pulling from the storyboard)